_about/notes/0.8.md

```diff
@@ -8,4 +8,55 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model. There is a conversion tool built into `istioctl` to help migrate your config from
+the old model. [Learn more about the new traffic management model]({{home}}/docs/tasks/traffic-management-v1alpha3/).
+
+- **Envoy V2**. Users can choose to inject Envoy V2 as the sidecar. In this mode, Pilot uses the V2 Envoy API to program the proxies
+in the mesh, which increases effective scalability and should eliminate spurious 404 errors. [TBD: docs on how to control this?]
+
+- **Constrain Inbound Ports**. Restrict the inbound ports in a pod to the ones needed by the apps running inside that pod.
+[TBD: docs on this feature?]
+
+## Security
+
+- **Introducing Citadel**. We've finally given a name to our security component. What was
+formerly known as Istio-Auth or Istio-CA is now called Citadel.
+
+- **Multicluster Support**. We support per-cluster Citadel in multicluster deployments such that all Citadels share the same root cert
+and workloads can authenticate each other across the mesh.
+
+- **Authentication Policy**. We've introduced authentication policy that can be used to configure service-to-service
+authentication (mutual TLS) and end user authentication. This is the recommended way for enabling mutual TLS
+(over the existing config flag and service annotations). [TBD: doc link?]
+
+## Telemetry
+
+- **Self-Reporting**. Mixer and Pilot now produce telemetry that flows through the normal
+Istio telemetry pipeline, just like services in the mesh.
```

Comment:
I would not bring Pilot here, since Envoy's gRPC APIs are over a stream and there is not much telemetry here.

Reply:
We should mention it and keep it flowing (not remove the envoy in front of pilot)

Reply:
I think keeping Pilot here is fine, as the gRPC streaming is with opt-in v2, and even then there is some telemetry provided (same as any other gRPC streaming service in the mesh).

Reply:
Yes but that’s not what people expect. It will be one metric per Envoy instead of metrics for various requests inside ADS. Will reword it later.

Reply:
Should it be "route rules" instead of "routing rules"? Or "routing rules" are OK too?